File: in_Python/0070 Climbing Stairs.py
class Solution(object):
    def climbStairs(self, n):
        """
        :type n: int
        :rtype: int
        """
        temp = []
        temp.append(1)
        temp.append(1)
        for i in range(2,n+1):
            temp.append(temp[i-1]+temp[i-2])
        return temp[-1]

File: in_Java/0070 Climbing Stairs.java
class Solution {
    public int climbStairs(int n) {
        int[] t = new int[n+1];
        t[0] = 1;
        t[1] = 1;
        for(int i = 2; i <= n; i++){
            t[i] = t[i-2] + t[i-1];
        }
        return t[n];
    }
}